src/App.js

```diff
@@ -5,13 +5,16 @@ import MainPage from './components/MainPage';
 import DeleteTours from './components/DeleteTours';
 import TourDetails from './components/TourDetails';
 import CreateTour from './components/CreateTour';
+import LogInPage from './components/LogInPage';
+
 import './styles/index.css';
 
 function App() {
   return (
     <>
       <Nav />
       <Routes>
+        <Route path="/LogInPage" element={<LogInPage />} />
```

Comment:
`OPTIONAL` Please consider changing the route to just `/login`. It would be more precise.